src/saving_tools.jl

```diff
@@ -112,8 +112,8 @@ function read_stdout_stderr(cmd::Cmd)
     return (exception = exception, out=read(out,String), err=read(err,String))
 end
 
-"""
-    gitpatch(gitpath = projectdir())
+""" 
+gitpatch(gitpath = projectdir())
```

Comment:
Incorrect. Restore the 4 spaces of identation, otherwise the documentation string will not register correctly.